### DISCUSSION TITLE
Add a link in the UI to the latest changelog

### DISCUSSION THREAD

**Discussion by sbts** [ledgersmb/LedgerSMB#5851] [Ideas]
Somewhere on the "info" screen shown on successful login we should add a prominent link to the changelog.

This can be done reasonably easily by pointing at one of
- https://github.com/ledgersmb/LedgerSMB/blob/master/Changelog (to include all changelog entries, including for future stable release)
- https://github.com/ledgersmb/LedgerSMB/blob/1.7/Changelog  (for the 1.7 series)

Use of information from
- https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#section-links 

Should allow us to add section links (a TOC) to the top of the Changelog file making navigation to individual versions easier for the user. 

The viewing experience for the user can be further enhanced by renaming
`Changelog` to `Changelog.md` so that github renders the content as markdown automatically

Adding this may assist users to decide if they want to immediately upgrade, or defer until the next release.

This should probably be done in the 1.7 series so it's available to users considering upgrading to 1.8 when that's released later this year

**Comment by ehuelsmann** [comment]
But... Our changelog really isn't md.

**Comment by sbts** [comment]
True, it's not currently real markdown, but it is close.
To the point that making it markdown would have little impact on the way it appears in a raw form, but would significantly improve readability when viewed via a markdown engine

**Comment by ehuelsmann** [comment]
It's rather unclear how to implement this request: Where do we point these links on master, so that the only step required on a release is to replace the links?!

**Comment by ehuelsmann** [comment]
My point on the previous post is that we already have a tremendous list of documents which need to be (manually) adjusted when a new release branch is being created. I'm very much against creating more manual steps in the process to create a new release branch.

**Comment by neilt** [comment]
A lot of current LSMB users are also installers, but I think that fact should NOT be the basis of LSMB system design.  Users are not interested in the change log, installers are.  I've never used an accounting system with a prominent link to the change log on the login screen. It will just be confusing to normal users.

In order for the ChangeLog to be useful to users its scope would have to change from "What changed, high level summary?" to "How does the change impact the user?".  These are two different documents with the latter normally generated by the install team and focused on the actual installation and user needs.

So I am not in favor of providing a link to the change log particularly when it requires additional work effort the development team does not have.

However, I can see value in having a link to a generic file or database entry that the installer can edit and maintain outside of LSMB and is easily viewable by the users. Maybe not a link, but a permanent display after login or a News menu item. If the installer wants to copy the current release notes into this file, then they are free to do so. The responsibility for this content should be the local installer. 

I do not think this News functionality should be a priority right now because it could just as easily be a locally written document distributed to each user after an upgrade.

**Reply by ehuelsmann** [reply]
Agreed. Which would be possible if we solve #9196 . Marking as closed and referring there.

**Comment by ehuelsmann** [comment]
Duplicate of #9196
